blocks/log.py

```diff
@@ -64,10 +69,3 @@ def previous_row(self):
     @property
     def last_epoch_row(self):
         return self[self.status['_epoch_ends'][-1]]
-
-    def to_dataframe(self):
-        """Convert a log into a :class:`.DataFrame`."""
-        if not PANDAS_AVAILABLE:
-            raise ImportError("The pandas library is not found. You can"
-                              " install it with pip.")
-        return DataFrame.from_dict(self, orient='index')
```

Comment:
I don't think we should have removed that.

For one, now the user has to remember this class and typo more, and `to_dataframe` is often used in an interactive setting (and before it is done essentially nothing can be done with the log). In addition when we have more then one backend (and I believe we should), this method could hide from the user the most efficient way to convert a log to a dataframe. Also `blocks-plot` is probably broken now, as far as I remember `to_dataframe` was used there.